src/main/java/org/opentripplanner/updater/GraphUpdaterConfigurator.java

```diff
@@ -61,72 +64,75 @@ public static void setupGraph(Graph graph, JsonNode mainConfig) {
+                            break;
+                        case "example-polling-updater":
+                            updater = new ExamplePollingGraphUpdater();
+                            ((ExamplePollingGraphUpdater) updater).configure(graph, configItem);
+                            break;
+                        case "winkki-polling-updater":
+                            updater = new WinkkiPollingGraphUpdater();
+                            ((WinkkiPollingGraphUpdater) updater).configure(graph, configItem);
+                            break;
+                        case "siri-et-updater":
+                            updater = new SiriETUpdater();
+                            ((SiriETUpdater) updater).configure(graph, configItem);
+                            break;
+                        case "siri-vm-updater":
+                            updater = new SiriVMUpdater();
+                            ((SiriVMUpdater) updater).configure(graph, configItem);
+                            break;
+                        case "siri-sx-updater":
+                            updater = new SiriSXUpdater();
+                            ((SiriSXUpdater) updater).configure(graph, configItem);
+                            break;
+                    }
                 }
-                else if (type.equals("siri-sx-updater")) {
-                    updater = new SiriSXUpdater();
+                if (updater != null) {
+                    updaters.add(updater);
                 }
             }
-
-            if (updater == null) {
-                LOG.error("Unknown updater type: " + type);
-            } else {
-                try {
-                    // Inform the GraphUpdater of its parent Manager so the updater can enqueue write operations.
-                    // Perhaps this should be done in "addUpdater" below, to ensure the link is reciprocal.
-                    updater.setGraphUpdaterManager(updaterManager);
-                    // All GraphUpdaters are JsonConfigurable - send them their config information.
-                    updater.configure(graph, configItem);
-                    // Perform any initial setup in a single-threaded manner to avoid concurrent reads/writes.
-                    updater.setup(graph);
-                    // Add graph updater to manager.
-                    updaterManager.addUpdater(updater);
-                    LOG.info("Configured GraphUpdater: {}", updater);
-                } catch (Exception e) {
-                    LOG.error("Failed to configure graph updater:" + configItem.asText(), e);
-                }
+            catch (Exception e) {
+                LOG.error("Failed to configure graph updater:" + configItem.getType(), e);
             }
         }
-        // Now that all the updaters are configured, kick them all off in their own threads.
-        updaterManager.startUpdaters();
-        return updaterManager;
```

Comment:
I cannot see that the updater are started in the new code?

Reply:
The updaters are started in GraphUpdaterConfigurator.java line 51.